cmd/atlantis_generate_repo_config.go

```diff
@@ -36,35 +36,35 @@ func init() {
 
 	atlantisGenerateRepoConfigCmd.PersistentFlags().String("stacks", "",
 		"Generate Atlantis projects for the specified stacks only (comma-separated values).\n"+
-			"atmos atlantis generate repo-config --config-template <config_template> --project-template <project_template> --stacks <stack1>,<stack2>\n"+
+			"atmos atlantis generate repo-config --config-template &ltconfig_template&gt --project-template &ltproject_template&gt --stacks &ltstack1&gt,&ltstack2&gt\n"+
 			"The filter can contain the names of the top-level stack manifests and the logical stack names (derived from the context vars)\n"+
-			"atmos atlantis generate repo-config --config-template <config_template> --project-template <project_template> --stacks orgs/cp/tenant1/staging/us-east-2,orgs/cp/tenant2/dev/us-east-2\n"+
-			"atmos atlantis generate repo-config --config-template <config_template> --project-template <project_template> --stacks tenant1-ue2-staging,tenant1-ue2-prod\n"+
-			"atmos atlantis generate repo-config --config-template <config_template> --project-template <project_template> --stacks orgs/cp/tenant1/staging/us-east-2,tenant1-ue2-prod",
+			"atmos atlantis generate repo-config --config-template &ltconfig_template&gt --project-template &ltproject_template&gt --stacks orgs/cp/tenant1/staging/us-east-2,orgs/cp/tenant2/dev/us-east-2\n"+
+			"atmos atlantis generate repo-config --config-template &ltconfig_template&gt --project-template &ltproject_template&gt --stacks tenant1-ue2-staging,tenant1-ue2-prod\n"+
+			"atmos atlantis generate repo-config --config-template &ltconfig_template&gt --project-template &ltproject_template&gt --stacks orgs/cp/tenant1/staging/us-east-2,tenant1-ue2-prod",
```

Comment:
- `&lt` and `&gt` is not even valid HTML ;) (it's `&lt;` and `&gt;`)
- The whole point of this PR is to use markdown syntax, especially code fences and back ticks
- 
```suggestion
			"`atmos atlantis generate repo-config --config-template <config_template> --project-template <project_template> --stacks orgs/cp/tenant1/staging/us-east-2,orgs/cp/tenant2/dev/us-east-2`\n"+
			"`atmos atlantis generate repo-config --config-template <config_template> --project-template <project_template> --stacks tenant1-ue2-staging,tenant1-ue2-prod`\n"+
			"`atmos atlantis generate repo-config --config-template <config_template> --project-template <project_template> --stacks orgs/cp/tenant1/staging/us-east-2,tenant1-ue2-prod`",
```